src/writer/serve.py

```
@@ -31,7 +32,7 @@
 from urllib.parse import urlsplit
 
 import uvicorn
-from fastapi import Depends, FastAPI, HTTPException, Request, Response
+from fastapi import Depends, FastAPI, File, HTTPException, Request, Response, UploadFile
```

Flake8 (7.2.0)

[error] 'fastapi.Depends' imported but unused

(F401)